lib/universal-search.js

```diff
@@ -88,22 +88,47 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
 
+    //Const for main stylesheet.
     const stylesheet = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
-    stylesheet.rel = 'stylesheet';
     stylesheet.href = 'chrome://universalsearch-skin/content/style.css';
+    stylesheet.rel = 'stylesheet';
     stylesheet.type = 'text/css';
     stylesheet.id = 'universal-search-stylesheet';
     stylesheet.style.display = 'none';
 
+    //Const for platform-specific stylesheet.
```

Comment:
And here, I'd remove the individual comments above the conditional branches below (lines 109, 112, 115), and instead of the comment on line 102, say something like:

```
// Insert platform-specific styles, using `Services.appinfo.OS` to
// distinguish between Mac OS ('Darwin'), Linux, and Windows ('WINNT').